paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -9,6 +9,106 @@
     <script src="https://{{cdnDomain}}/assets/cmp/cmp.js"></script>
     <script>CMP.cmpCheck();</script>
 
+    <!-- Prebid.js -->
+    <script async src="//www.googletagservices.com/tag/js/gpt.js"></script>
+    <script async src="/assets/publisher/prebid/prebid.js"></script>
```

Comment:
[OPT] We might want to move prebid.js to the cdn domain for consistency, but this can be done later anyway